### PR TITLE
chore: Fix column name in proxy ls command

### PR DESCRIPTION
**Pull request by Emyrk**
```
NAME              URL                           PROXY STATUS  
brazil-saopaulo   https://brazil.dev.coder.com  ok            
europe-frankfurt  https://europe.dev.coder.com  ok            
sydney            https://sydney.dev.coder.com  ok 
```

### CODE REVIEW

**Review by bpmct — COMMENTED** [review]
Does it have to be called proxy status? To me, `Status` is more natural. It feels inconsistent. Wouldn't all be `proxy name`, `proxy url`, `proxy status`?

**Review by bpmct — APPROVED** [review]

**Comment by Emyrk** [comment]
@bpmct I lokoed into removing the word `Proxy` to make it just `Status`, but it requires a lot more code changes.

It gets a bit tricky because the recursive nesting naming is how we uniquely identify some struct field. So I need to add another tag to indicate ignore the parent name, or pass down some options...

tl;dr it gets more tricky. And maybe it is worth it, but for now this makes things better and I can spend the time to make this `Status` when there is less pressing things imo. I'll have spare time next week to take on these more minor tasks.

**Comment by bpmct** [comment]
Ah, sorry I didn't respond. Makes sense. There are bigger fish to fry
